csharp-basic-topics/ValidateNumberIsPositiveOrNegative/ValidateNumberIsPositiveOrNegative/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ValidateNumberIsPositiveOrNegative;
+
+public class Program
+{
+    private static void Main(string[] args)
+    {
+        var number = -42;
+        Console.WriteLine($"The function will return 1 if number is greater than zero, -1 if the number is less than zero, 0 if the number is equal to zero.");
+        Console.WriteLine($"IsPositiveOrNegative Using Conditional Statement -> {NumberValidation.IsPositiveOrNegativeUsingConditionalMethod(number)} ");
+        Console.WriteLine($"IsPositiveOrNegative Using Bitwise Oerator RightShift -> {NumberValidation.IsPositiveOrNegativeUsingRightShiftMethod(number)} ");
```

Comment:
typo in Operator word. Both for left and right shift

Reply:
corrected.